README.md

```diff
@@ -363,6 +363,7 @@ These can be useful for osint and social engineering.
 - [Yandex Image](https://www.yandex.com/images/) - Yandex Image Search
 - [Facecheck.id](https://facecheck.id/) - Search for people by photo and verify you are talking to the person they claim to be
 - [Bing Visual Search](https://www.bing.com/visualsearch) - See it, search it
+- [Tineye Search](https://tineye.com/) - Reverse Image Search
```

Comment:
This should be:
```
- [Tineye](https://tineye.com/) - Image search and recognition company
```

Reply:
@edoardottt thanks. I have worked on this

Reply:
here you're trying to add another search engine, please correct that + conflicting readme

Reply:
I have fixed this @edoardottt